test/tests.py

```diff
@@ -0,0 +1,60 @@
+from operations import Operations
+from http import HTTPStatus
+import unittest
+import os
+
+# This code is probably existing as a placeholder only for testing purposes,
+# final code here will be using comparator's results. Also, won't be sending request directly to target cluster,
+# and will be using the replayer instead, so we're able to check comparator's results once the replayer outputs
+# the triples log file.
+# TODO: Add endpoints to be read from environment variable.
+
+
+class MyTestCase(unittest.TestCase):
+    def test_index(self):
+        endpoint1 = os.getenv('ENDPOINT_1', 'http://localhost:9200')
```

Comment:
In general, it isn't the best idea to pull test configuration during time-of-test; it's possible that the environment variable/files may change, for example, especially on long-running test suites.  It also makes is hard for maintainers to reason about the boundaries of these tests - they have to look at each and every test to figure out whether the test needs something from outside the pytest process (whether a file, an environment variable, etc).  For these reasons, I recommend pulling your out-of-process configuration for your tests in a single, unified location and then letting your tests refer to the in-memory state created by that piece of code.

Here's an example of how you can do that using pytest fixtures:
https://automationpanda.com/2023/01/12/passing-test-inputs-into-pytest/